network-api/networkapi/wagtailpages/rss.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+from django.contrib.syndication.views import Feed
+from django.utils.feedgenerator import Atom1Feed
+
+from bs4 import BeautifulSoup
+
+from .models import IndexPage
+
+
+class RSSFeed(Feed):
+    """
+    Blog page RSS feed, using the content:encoded serializer above
+    """
+
+    title = 'Mozilla Foundation Blog'
+    link = 'https://foundation.mozilla.org/blog/'
```

Comment:
because the Atom feed will turn that into `example.com/blog` for reasons that no one seems to be able to explain.

Reply:
It uses whatever is set in "Sites" as the domain for the site id. So if you have that set to `http://example.com`, it will use that as a domain.

Reply:
Right, for Django. But Wagtail, which often does the same thing, in this case doesn't: rather than `SITES`, Wagtail handles site/domain management in the database. So I'd rather keep things hardcoded in the rss file to make it immediately obvious what needs to be changed if we do need to change something, rather than adding a variable that effectively "bypasses" the Wagtail way of doing things, and will most likely not be found as being the reason for things not working when a future dev does some googling to figure out what to change.